docs/source/guides/index.mdx

```diff
@@ -186,6 +186,56 @@ introspection:
 apollo-mcp-server <path to the preceding config>
 ```
 
+## Authorization
+
+The Apollo MCP server supports authorizing clients (e.g. LLMs) in accordance with [the MCP specification](https://modelcontextprotocol.io/specification/2025-06-18/basic/authorization).
+
+To set up auth in the MCP server, you will need:
+1. An [OAuth 2.1-compliant](https://oauth.net/2.1/) Identity Provider (for example, Auth0, Okta, Keycloak, are popular IdPs).
+1. An Apollo Router [configured to use JWT authentication](/graphos/routing/security/jwt).
+1. A configuration file for the MCP server that [defines the `auth` settings](/apollo-mcp-server/command-reference#auth).
+
+### Example auth configuration with Auth0
```

Comment:
FYI @jcaudle is working through a more detailed guide on this that folks can actually follow along with. It needs more time to run through though, so in the meantime I thought this section might help. Please let me know if it's too confusing and we can just keep it generic with the steps above.